test/spec/Feature/Query/RpcSpec.hs

```diff
@@ -1452,17 +1452,29 @@ spec actualPgVersion =
           resHeaders `shouldSatisfy` elem ("X-Header", "str")
           resBody `shouldBe` [json|{"code":"123","message":"ABC","details":null,"hint":null}|]
 
-      it "returns error for invalid JSON in RAISE Message field" $
+      it "returns error for invalid JSON in the MESSAGE option of the RAISE statement" $
         get "/rpc/raise_sqlstate_invalid_json_message" `shouldRespondWith`
-          [json|{"code":"PGRST121","message":"The message and detail field of RAISE 'PGRST' error expects JSON","details":null,"hint":null}|]
+          [json|{
+            "code":"PGRST121",
+            "message":"Could not parse JSON in the \"RAISE SQLSTATE 'PGRST'\" error",
+            "details":"Invalid JSON value for MESSAGE: 'INVALID'",
+            "hint":"MESSAGE must be a JSON object with obligatory keys: 'code', 'message' and optional keys: 'details', 'hint'."}|]
           { matchStatus = 500 }
```

Comment:
@laurenceisla Btw, the docs show [PGRST121](https://postgrest.org/en/latest/references/errors.html#pgrst121) as a 400

Reply:
Thanks for the heads up, fixed in https://github.com/PostgREST/postgrest/commit/85a4e35a2de869ddf9b6c5f6893a3154767cbc76.